### PR TITLE
SER-93 | Testing 

### PR DESCRIPTION
**Pull request by SongRaein**
SER-93 | Testing | Ad hoc

### CODE REVIEW

File: SnowTesting.txt [hunk 1]
```diff
@@ -1,2 +1,4 @@
 Once user selects a business service with the "Ticket" as the call type and clicks on update button it will redirects to the service gateway page. Then application will fetch the TKT details based on the "tktsysid" provided in the url and populates the description, short description and the requester name in the service gateway.
 Changed
+
+change again!!
```
Comment (lhansamali) [inline SnowTesting.txt:4]: In progress!!

**Review by lhansamali — COMMENTED** [review]
created

File: SnowTesting.txt (hunk 1 above)
Comment (SongRaein) [inline SnowTesting.txt:4]: Corrected

**Review by SongRaein — COMMENTED** [review]
Finish

**Review by lhansamali — APPROVED** [review]

**Review by SongRaein — COMMENTED** [review]
Added

File: SnowTesting.txt [hunk 2]
```diff
@@ -1,2 +1,4 @@
 Once user selects a business service with the "Ticket" as the call type and clicks on update button it will redirects to the service gateway page. Then application will fetch the TKT details based on the "tktsysid" provided in the url and populates the description, short description and the requester name in the service gateway.
 Changed
+
```
Comment (SongRaein) [inline SnowTesting.txt:3]: Started

**Review by SongRaein — COMMENTED** [review]
Finish RW

**Pull request merged**